src/proposals/mips/mip-b19/MIP-B19.md

```diff
@@ -0,0 +1,5 @@
+# MIP-B19: Safety Module Reward Speed Adjustment
```

Comment:
Proposal should be MIP-B19. I think the reason why the directory/filenames are off is that Gauntlet included MIP-B19 in the files for MIP-M29. We might want to ask them to keep this separate to avoid name conflicts going forward.